config/printer-ratrig-v-minion-2021.cfg

```diff
@@ -0,0 +1,178 @@
+# This file contains pin mappings for the Ratrig V-Minion
+# with an Octopus Pro v1.1 board.
+#
+# To use this config, during "make menuconfig" select the STM32F446
+# with a "32KiB bootloader" and USB (on PA11/PA12)
+# communication.
```

Comment:
Please state that this F446 board needs a 12MHZ crystal(stock crystal setting causes many timing issues

Thanks
James